### ISSUE TITLE
Auto Slipping of Errors

### ISSUE THREAD

**Issue by sterelok123** [Bionus/imgbrd-grabber#1634]
Hi,

I suggest you add an option to automatically skip all files that cannot be downloaded for a smooth downloading experience. It gives an error every-so-often and I'd have to skip it manually which is rather annoying. 

Thank you.

Best regards,

**Comment by esotericer1** [comment]
Yeah, I second that opinion. I have that problem with zerochan files.

**Comment by Atulin** [comment]
+1, some custom sources I added can be problematic with some larger files, for example. Would be cool if those just got auto-skipped instead of me having to mash the skip button.

**Comment by Bionus** [comment]
So from what I understand, if an image download never completes for some reason, it should be skipped and marked as failed automatically? (for example if no progress on a download for more than 30s or something)

How would it be differentiated from a normal network issue? (wi-fi lost or similar)

**Comment by Atulin** [comment]
Yes, just automatically marked as failed, or maybe pushed to the end of the queue for redownload once or twice.

I'd imagine figuring out if it's a failure to download or a network issue would be nontrivial, so I'd say just let the images keep failing if network is lost. The user can always retry them.
